projects/plugins/jetpack/modules/widgets/internet-defense-league.php

```diff
@@ -141,29 +204,57 @@ public function form( $instance ) {
 		$this->select( 'badge', $this->badges, $instance['badge'] );
 		echo '</label></p>';
 
-		/* translators: %s is a name of an internet campaign called the "Internet Defense League" */
-		echo '<p>' . sprintf( _x( 'Learn more about the %s', 'the Internet Defense League', 'jetpack' ), '<a href="https://www.internetdefenseleague.org/">Internet Defense League</a>' ) . '</p>';
+		echo '<p>' . wp_kses(
+			sprintf(
+				/* translators: %s is a name of an internet campaign called the "Internet Defense League" */
```

Comment:
Not just a name. HTML linking to a website.
```suggestion
				/* translators: %s is an HTML link to the website of an internet campaign called the "Internet Defense League" */
```